x-pack/legacy/plugins/maps/public/connected_components/map/mb/tooltip_control/tooltip_control.js

```diff
@@ -277,14 +277,48 @@ export class TooltipControl extends React.Component {
     });
   };
 
+  _getLayerName = async (layerId) => {
+    const layer = this._findLayerById(layerId);
+    if (!layer) {
+      return null;
+    }
+
+    return layer.getDisplayName();
+  }
+
+  _renderTooltipContent = () => {
+    const publicProps = {
+      addFilters: this.props.addFilters,
+      closeTooltip: this.props.clearTooltipState,
+      features: this.props.tooltipState.features,
+      isLocked: this.props.tooltipState.type === TOOLTIP_TYPE.LOCKED,
+      loadFeatureProperties: this._loadFeatureProperties,
+      loadFeatureGeometry: this._loadFeatureGeometry,
+      getLayerName: this._getLayerName,
+    };
+
+    if (this.props.renderTooltipContent) {
```

Comment:
Small thing, but I wonder if we should consider setting default props to `false` on `renderTooltipContent`

Reply:
I do not understand what you mean. Could you elaborate?

Reply:
Actually `null` would be more appropriate here. I think it just feels strange to rely on a prop that's `undefined` embedded several components down without having a default value. My first thought was, "we're relying on this value, what's the default if not overridden?" But yeah,  `undefined` evaluates to `false` so it doesn't really make a difference.